src/core/src/package_managers/ZypperPackageManager.py

```diff
@@ -172,7 +175,7 @@ def __handle_zypper_updated_or_reboot_exit_codes(self, command, code):
                 str(code), command))
             return
 
-        if code == self.zypper_exitcode_zypper_updated:
+        if code == self.zypper_exitcode_zypper_updated or self.zypper_out_zypper_updated_msg in out:
             self.composite_logger.log_debug(
                 " - Package manager update detected. Patch installation run will be repeated.")
```

Comment:
Update log to One of the installed patches affects the package manager. Patch installation run will be repeated